app/src/main/java/com/android/sample/ui/profile/EditProfile.kt

```diff
@@ -122,17 +122,7 @@ fun EditProfileScreen(
         if (isGalleryOpen) {
           GalleryScreen(
               isGalleryOpen = { isGalleryOpen = false },
```

Comment:
Updated GalleryScreen and ProfileImage composables to handle image editing and removal.